src/pam_pgsql.h

```diff
@@ -19,9 +19,9 @@
 #define PAM_SM_PASSWORD
 #include <security/pam_modules.h>
 
-#define PASSWORD_PROMPT         "Password: "
-#define PASSWORD_PROMPT_NEW	    "New password: "
-#define PASSWORD_PROMPT_CONFIRM "Confirm new password: "
+#define PASSWORD_PROMPT         "DATABASE password: "
+#define PASSWORD_PROMPT_NEW	    "New DATABASE password: "
+#define PASSWORD_PROMPT_CONFIRM "Confirm new DATABASE password: "
 
```

Comment:
Reverted, I wanted to add option similar to authtok_type in [pam_cracklib](https://linux.die.net/man/8/pam_cracklib) but has not finish it yet. I do it in a new branch.